wetterdienst/provider/dwd/metadata/constants.py

```diff
@@ -13,6 +13,7 @@
 
 class DWDCDCBase(Enum):
     CLIMATE_OBSERVATIONS = "observations_germany/climate/"
+    CLIMATE_URBAN_OBSERVATIONS = "observations_germany/climate_urban/"
 
 
 DWD_FOLDER_MAIN = "./dwd_data"
```

Comment:
Maybe use pathlib here

Reply:
This is actually an artefact that should be removed as we don't store data anymore locally after downloading. Thanks for spotting it!